clang/lib/AST/StmtPrinter.cpp

```diff
@@ -630,6 +630,10 @@ void StmtPrinter::VisitObjCAvailabilityCheckExpr(
   OS << "@available(...)";
 }
 
+void StmtPrinter::VisitObjCFeatureCheckExpr(ObjCFeatureCheckExpr *Node) {
+  OS << "@feature(...)";
```

Comment:
I think the body was elided on `@available(...)` because the AST doesn't retain full fidelity of the availability check and instead it drops the other platforms. Do we have that same problem with `@feature()`? If not, I think this should print the actual feature too, so you can write a test that checks it round-trips through serialization.

Reply:
I've changed this to use `@available` instead of introducing a new keyword but haven't made any changes to `StmtPrinter` yet.